Crashing on XMPP with a panic

Running 1.22.3 my process started crashing today every few minutes with this:

```
panic: runtime error: index out of range [0] with length 0

goroutine 738 [running]:
github.com/matterbridge/go-xmpp.(*Client).Recv(0xc0001da6c0, 0xc000b71a40, 0x20107fe, 0x11, 0xc000b71908)
        /go/src/github.com/42wim/matterbridge/vendor/github.com/matterbridge/go-xmpp/xmpp.go:794 +0x23ac
github.com/42wim/matterbridge/bridge/xmpp.(*Bxmpp).handleXMPP(0xc0001309c0, 0x0, 0x0)
        /go/src/github.com/42wim/matterbridge/bridge/xmpp/xmpp.go:275 +0xcc
github.com/42wim/matterbridge/bridge/xmpp.(*Bxmpp).manageConnection(0xc0001309c0)
        /go/src/github.com/42wim/matterbridge/bridge/xmpp/xmpp.go:225 +0x27f
created by github.com/42wim/matterbridge/bridge/xmpp.(*Bxmpp).Connect
        /go/src/github.com/42wim/matterbridge/bridge/xmpp/xmpp.go:52 +0x22a
```

I made no changes today.

--- a/bridge/xmpp/xmpp.go
+++ b/bridge/xmpp/xmpp.go
@@ -283,9 +283,9 @@ func (b *Bxmpp) handleXMPP() error {
 	for {
 		m, err := b.xc.Recv()
 		if err != nil {
+			// An error together with AvatarData is non-fatal
 			switch m.(type) {
 			case xmpp.AvatarData:
-				b.avatarAvailability[v.From] = false
 				continue
 			default:
 				return err

--- a/bridge/xmpp/xmpp.go
+++ b/bridge/xmpp/xmpp.go
@@ -283,7 +283,13 @@ func (b *Bxmpp) handleXMPP() error {
 	for {
 		m, err := b.xc.Recv()
 		if err != nil {
-			return err
+			switch m.(type) {
+			case xmpp.AvatarData:
+				b.avatarAvailability[v.From] = false
+				continue
+			default:
+				return err
+			}
 		}
 
 		switch v := m.(type) {

